config/filter.d/common.conf

```diff
@@ -83,7 +83,7 @@ datepattern = %(lt_short/datepattern)s
 [lt_rfc5424]
 # RFC 5424 log-format, see gh-2309:
 #__prefix_line = \s*<__hostname> <__daemon_re> \d+ \S+ \S+\s+
-__prefix_line = \s*<__hostname> <__daemon_re> \d+ \S+ (?:[^\[\]\s]+|(?:\[(?:[^\]"]*|"[^"]*")*\])+)\s+
+__prefix_line = \s*<__hostname> <__daemon_re> [\d+-] \S+ (?:[^\[\]\s]+|(?:\[(?:[^\]"]*|"[^"]*")*\])+)\s+
```

Comment:
This is not correct RE, see https://github.com/fail2ban/fail2ban/issues/2711#issuecomment-623976515